crates/js-sys/src/lib.rs

```diff
@@ -2835,6 +2835,7 @@ macro_rules! number_try_from {
             #[inline]
             fn try_from(x: $x) -> Result<Number, Self::Error> {
                 let x_f64 = x as f64;
+                #[allow(clippy::manual_range_contains)]
                 if x_f64 >= Number::MIN_SAFE_INTEGER && x_f64 <= Number::MAX_SAFE_INTEGER {
```

Comment:
This errored when reorganizing test runs in the CI, I don't know why it doesn't error normally.

Reply:
I'm assuming this doesn't affect the CI run of this PR?
If that's the case, could you address it when we actually get there?

I was already working separately on making sure Clippy runs properly on CI. Currently the Clippy run on CI skips a lot of code that otherwise would throw warnings, that should definitely be addressed.

Reply:
I was wondering about that since I saw other clippy warnings that didn't error in the CI either, I just happened to run into this one. This PR doesn't need it.